pygmt/src/plot.py

```diff
@@ -144,11 +144,13 @@ def plot(self, x=None, y=None, data=None, sizes=None, direction=None, **kwargs):
         the coordinates of a *refpoint* which will serve as a fixed external
         reference point for all groups.
     {G}
-    intensity : float or bool
-        Provide an *intens* value (nominally in the -1 to +1 range) to
-        modulate the fill color by simulating illumination [None]. If
-        using ``intensity=True``, we will instead read *intens* from the
-        first data column after the symbol parameters (if given).
+    intensity : float or bool or 1d array
+        Provide an *intensity* value (nominally in the -1 to +1 range) to
+        modulate the fill color by simulating illumination. If using
```

Comment:
```suggestion
        modulate the color fill by simulating illumination. If using
```

Reply:
Not sure, maybe both versions work. Otherwise the PR looks fine to me, great work @core-man.

Reply:
GMT plot uses fill color, see https://docs.generic-mapping-tools.org/latest/plot.html#i. So let's keep the original version.